spec/support/service_account_authorization_helper.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module ServiceAccountAuthorizationHelper
+  def service_account_authorization_header(scope:)
```

Comment:
💡 We can probably refactor the other specs that stub the service account authentication for the controller to instead send the authorization header I've implemented as a helper here.